### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by dananhsl**
Hey! I worked on the Home and About section a bit. Please take a look and inform me about any bugs you find.

**Didn't work on Portfolio, contact or footer yet, changes soon to come.**
**Contact Button in Navbar not working! I am aware of the issue and working on it.**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -3,7 +3,7 @@
     margin: 0;
     padding: 0;
 }
-
+/* ---------------------------- BODY ----------------------------*/
```
Comment (p-enz) [inline css/style.css:6]: Like style of your comments, very good to read

**Review by p-enz — COMMENTED** [review]
Looks good to me, well done :)

File: css/style.css (hunk 1 above)
Comment (dananhsl) [reply css/style.css:6]: Thanks m8! :D

**Comment by lkk95** [comment]
Really nice website! I like the background :)

**Pull request merged**